templates/.travis.yml

```diff
@@ -114,12 +114,53 @@ jobs:
       - COPY="all the environment settings from your job"
 
   include:
+(??)    # libstdc++
```

Comment:
I'm curious what does this `(??)` trick do?

Reply:
WTF? Nice conflict resolution :D

Reply:
It happened to me, but I committed the `<<<` markup :-)
I honestly did not know this `(??)` variant.

Reply:
Not sure. Another kind of conflict